pkg/airgap/config.go

```diff
@@ -0,0 +1,11 @@
+package airgap
+
+import "github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+
+func SetAirgapConfig(cfg *v1beta1.ClusterConfig) {
```

Comment:
Can we have this function to live inside config package instead ?

Reply:
I'm pretty ambivalent on this - does it fit better as part of airgap or as part of config?